cmd/acra-connector/acra-connector.go

```diff
@@ -230,13 +230,14 @@ func main() {
 	tlsAcraserverSNI := flag.String("tls_acraserver_sni", "", "Expected Server Name (SNI) from AcraServer")
 	tlsAuthType := flag.Int("tls_auth", int(tls.RequireAndVerifyClientCert), "Set authentication mode that will be used in TLS connection with AcraServer/AcraTranslator. Values in range 0-4 that set auth type (https://golang.org/pkg/crypto/tls/#ClientAuthType). Default is tls.RequireAndVerifyClientCert")
 	tlsOcspURL := flag.String("tls_ocsp_url", "", "OCSP service URL")
-	tlsOcspRequired := flag.String("tls_ocsp_required", "denyUnknown", "Whether we need OCSP response in order to accept certificate")
-	tlsOcspFromCert := flag.String("tls_ocsp_from_cert", "prefer", "How should we treat OCSP server described in certificate itself")
+	tlsOcspRequired := flag.String("tls_ocsp_required", "denyUnknown", "How to treat certificates unknown to OCSP: <denyUnknown|allowUnknown|requireGood>")
+	tlsOcspFromCert := flag.String("tls_ocsp_from_cert", "prefer", "How should we treat OCSP server described in certificate itself; <use|trust|prefer|ignore>")
```

Comment:
I believe this one should be a full colon `:`, not semicolon `;`

Ditto in AcraServer.